images/01-base/Dockerfile

```diff
@@ -6,6 +6,8 @@ RUN rm /sbin/poweroff /sbin/reboot /sbin/halt && \
     echo 'RancherOS \n \l' > /etc/issue && \
     rm -rf /run \
        /linuxrc \
+       /etc/logrotate.conf \
+       /etc/logrotate.d/* \
```

Comment:
Was initially just for a bit of clean up - but unnecessary I guess. I've removed it.